### ISSUE TITLE
/Volunteers page needs work before it is pushed to prod

### ISSUE THREAD

**Issue by aronszanto** [zerobase-io/smart-tracing#261]
1. It is not discoverable unless you know exactly where to look
2. Needs significant copy editing from someone approved to sign off on web copy
3. Slack channel links will throw confusing errors if someone is not yet logged into our slack
4. Instructions as to how to join us need to be more clear, and maybe just the volunteer form is necessary. We can have a modal for more instructions, but right now it's really hard to follow what actually has to happen. In that modal, we should explain about NECSI workspace, etc and be up front that they need to follow *these exact instructions*.
5. It looks like we're only recruiting for technical volunteers, and that's not true. Consider not having slack channel information on the website (bad practice anyway, they can change much more frequently than the website does) and instead lay out our teams on the website and describe the kinds of people that work there. Some preliminary copy is right now on the GH.
6. There should be much more by way of exciting people to come use their superpowers for us. See GH for some inspiration.
7. Set up an automatic system to email volunteers information about getting set up in our slack when they sign up to take another swing at them.

**Comment by JohnCLo** [comment]
Sync up with @joshhopkins @eriksogn  to reach consensus and define sign off. It is precisely **not discoverable** and not elevated to a menu item or footer item, since I'm confident this page did not go through the necessary steps below I've defined below. If it has, then all stakeholders need to define who has final signoff.

I've made it pretty clear what should or should not be "marked as ready" to FE for this exact reason
https://necsi-edu.slack.com/archives/C010DL0BXKR/p1586261261412300?thread_ts=1586235849.408200&cid=C010DL0BXKR

Moving to duplicate.
